docs/diagrams/AddSequenceDiagram.puml

```diff
@@ -14,10 +14,10 @@ box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("add n/Google p/SWE hp/65025300002 \ne/support@google.com a/16 Park Way t/Urgent")
+[-> LogicManager : execute("add c/Google p/SWE \ne/support@google.com w/https://careers.google.com/ \n s/5000 Park Way t/Urgent")
```

Comment:
The `Salary` should be just `5000`, not `5000 Park Way`

Reply:
Thank you for the suggestion.